ops/internal/manage/depsets.go

```diff
@@ -156,8 +152,25 @@ func (dc *DepsetChecker) checkOnchain(cfgs []DiskChainConfig) error {
 		return nil
 	}
 
-	// Get and validate the first chain's addresses
-	firstAddrs, err := dc.getAndValidateAddresses(cfgs[0].Config.ChainID)
+	// Find chains that have already activated interop
+	now := time.Now().Unix()
+	var validChains []DiskChainConfig
```

Comment:
Is there a better name than `validChains`? I don't think it is invalid to not have interop activated yet.